### DISCUSSION TITLE
All photosphere (360 image) rooms suddenly won't load

### DISCUSSION THREAD

**Discussion by Aaron-E-Walsh** [mozilla/hubs#4671] [General Questions & Discussion] [answered]
Hello, we have our own Hubs Cloud instance running on AWS. I believe the last automatic Hubs Cloud update caused some problems, because now we cannot enter any of the photosphere (360 image) rooms that we created about 8 months ago.

These are very simple rooms, consisting of little more than a standard JPEG image that is set (I'm Spoke) to be 360 photosphere. Users spawn into the center of the 360 image upon entering these rooms.

We haven't entered these rooms in a while, but now that we are trying to use them they all present the same error message:

  THE SCENE FAILED TO LOAD

Although a "Reload" button is also shown it does not work.

We have confirmed that all odlf the images used to create these rooms are still at their original places online (where they were when the room were created in Spoke and working properly). So the image links are valid.

Is there any way to access these rooms, in Spoke or hubs, so that we can recover them in some way?

With thanks for your help!
Aaron

**Comment by Aaron-E-Walsh** [comment]
**e is not iterable** 

After further testing and investigating what might be causing the problem it appears that with the new update to Hubs Cloud there must be some minimal scene structure(s) that our Photosphere rooms do not have. When trying to re-publish these projects the error message **e is not iterable** is shown, and publishing fails.

To test this, I created a new blank project and left all the default content in it (floor plan, light, default crater mesh, spawn point). That publishes just fine, and so I added one of our photosphere images to it (and set projection to 360) and that version publishes just fine also. No problems.

BUT when I delete all of default content in this project (floor plan, crater mesh, etc.) and leave only the 360 photosphere in place it will not publish and the **e is not iterable** 

It seems that with the most recent update to Hubs Cloud update at least some core structural elements must be in place in order to publish a scene.

Is there any way for us to recover these photosphere rooms at this point? It took a lot of time and work for them to be constructed, and the individuals who created them will be distressed to find out they can't use them now (they filled the rooms up with content after we provided with with access, so they invested a lot of time completing these rooms after we published the bare-bones photospheres for them).

Perhaps there is a way to enable them even temporarily using browser development tools (to insert some code into the existing room that would give us access) so that we can at least go through and collect all of the content in each?

Any help you can give would be greatly appreciated!
Aaron

**Reply by pattersonbl2** [reply]
I am doing some digging into this question currently.  Are you still able to connect to your hubs instance at your domain ?

**Comment by Aaron-E-Walsh** [comment]
Thanks, Brandon. Yes, we are able to enter plenty other rooms on our hubs instance domain (just not most of our previously created 360 photosphere rooms).

**Comment by matthewbcool** [comment, marked as answer]
Spoke will not publish a scene without a model/ground plane:  https://github.com/mozilla/Spoke/issues/1162

There are baseline elements you need to publish or you will get this non-descriptive error:  "e is not iterable"

Try to include a model/groundplane.

**Comment by Aaron-E-Walsh** [comment]
Thank you, Matt. I did not realize that you had responded. My apologies for
the late reply.

I appreciate your advice, and we'll give that a shot.

Regards,
Aaron
